packages/core/src/lib/toast/toast-banner.svelte

```diff
@@ -20,7 +20,7 @@ export { extraClasses as cx };
 
 <div
   class={cx(
-    'relative flex w-auto items-center gap-2 border bg-medium p-2 text-sm',
```

Comment:
Neither `gap-2` (8px gap between flex children) nor `p-1` (4px padding on all sides) is correct, according to Figma:

<img width="828" alt="Screenshot 2023-12-26 at 09 05 58" src="https://github.com/viamrobotics/prime/assets/2963448/46ce2b91-58d7-4364-ab1b-3fdccb79357b">

- Left padding: 12px (`pl-3`)
- Right padding: 4px (`pr-1`)
- Top/bottom padding: none, use fixed height of 40px (`h-10`)

We're also missing a max-width of `480px` on this class

<img width="508" alt="Screenshot 2023-12-26 at 09 10 48" src="https://github.com/viamrobotics/prime/assets/2963448/5ee83dac-a751-4ce4-abd3-70eec75084df">